### PR TITLE
Add Kubernetes Helm Deployment chapter

### PR DESCRIPTION
**Pull request by jonatasbaldin**
## Description:
First draft of the Helm Chapter for the Kubernetes course, without metadata, I'll add them later.

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by RLutsch — APPROVED** [review]
lgtm, from a content perspective not sure about tilde

**Review by t-witchy — APPROVED** [review]

File: content/syllabuses/k8s.md [hunk 1]
```diff
@@ -33,4 +33,11 @@ title: Kubernetes (A.K.A K8s)
 
 - {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file/install-postgresql" >}}
 - {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file/deploy-frontend" >}}
-- {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file/deploy-backend" >}}
\ No newline at end of file
+- {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file/deploy-backend" >}}
+
+- {{< contentlink path="k8s/helm-deployment" >}}
```
Comment (sheenarbw) [inline content/syllabuses/k8s.md:38]: This topic is empty - should there be something in there? Or should it be excluded?

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Just one thing. We seem to be telling the learners to do a topic that is blank

File: content/syllabuses/k8s.md (hunk 1 above)
Comment (jonatasbaldin) [reply content/syllabuses/k8s.md:38]: It is supposed to just be a introductory text, which was missing. We might add more information there in the future.

**Pull request merged**